Dockerfile

```diff
@@ -1,38 +1,17 @@
-FROM ubuntu:14.04
-MAINTAINER Colin Alston <colin@praekelt.com>
-RUN apt-get update && apt-get -y --force-yes install libjpeg-dev zlib1g-dev libxslt1-dev libpq-dev nginx redis-server supervisor python-dev python-pip
-RUN apt-get -y install libffi-dev gettext
+FROM praekeltfoundation/django-bootstrap
+RUN apt-get-install.sh \
+    gettext \
+    libjpeg62 \
+    libtiff5
```

Comment:
I've just checked a built image and Pillow now includes all the libraries it needs to run in the wheel :). So this can just be:
```dockerfile
RUN apt-get-install.sh gettext
```